test/parallel/test-whatwg-url-parsing.js

```diff
@@ -22,6 +22,8 @@ const failureTests = tests.filter((test) => test.failure).concat([
   { input: null },
   { input: new Date() },
   { input: new RegExp() },
+  { input: 'test', base: null},
```

Comment:
nits: space is required before `}`. Also on line 26.

Reply:
Very concerned that the linter didn't fail locally - thanks!